lms/envs/common.py

```diff
@@ -3321,6 +3321,20 @@ def _make_locale_paths(settings):
 ]
 ENTERPRISE_CUSTOMER_COOKIE_NAME = 'enterprise_customer_uuid'
 BASE_COOKIE_DOMAIN = 'localhost'
+ENTERPRISE_LEARNER_ROLE = 'enterprise_learner'
+ENTERPRISE_ADMIN_ROLE = 'enterprise_admin'
+ENTERPRISE_OPERATOR_ROLE = 'enterprise_openedx_operator'
+ENTERPRISE_DASHBOARD_ADMIN_ROLE = 'dashboard_admin'
+ENTERPRISE_CATALOG_ADMIN_ROLE = 'catalog_admin'
+ENTERPRISE_ENROLLMENT_API_ADMIN_ROLE = 'enrollment_api_admin'
+SYSTEM_TO_FEATURE_ROLE_MAPPING = {
```

Comment:
It would be nice if we can import these constants from `edx-enterprise` instead of hardcoded values here.

Reply:
Done